app/components/zitting/manage-zittingsdata.hbs

```diff
@@ -1,5 +1,27 @@
-<div {{did-insert this.fetchBestuursorgaan}}>
-  {{#if this.showModal}}
+<div class="au-c-meeting-chrome-card au-u-margin-bottom-large">
```

Comment:
This content is now always shown, even when the modal is open. That prevents the background from jumping around when opening or closing the modal.